enhancements/20220720-alertmanager.md

```diff
@@ -0,0 +1,819 @@
+# Harvester supports `Alertmanager`
+
+Customers wish to enable Prometheus `Alertmanager` in Harvester with the ability to configure in UI and API interfaces.
+
+## Summary
+.
+
+### Related Issues
+
+https://github.com/harvester/harvester/issues/2517
+
+## Motivation
+
+`Alert` is the very important information for monitoring one running system. Customers can integrate it with other systems to have a well defined lifecycle management.
+
+### Goals
+
+Support `Alertmanager`.
+
+Support config various types of receivers.
+
+Support view and manage alerts from `Alertmanager`s dashboard.
+
+Does not support customized alert rules definition.
+
+Does not support customized template files to format alerts
+
+### Non-goals [optional]
+
+What is out of scope for this enhancement? Listing non-goals helps to focus discussion and make progress.
+
+## Proposal
+
+The Harvester `Alertmanager` will be achieved via `rancher-monitoring`.
+
+https://github.com/rancher/charts/tree/dev-v2.6/charts/rancher-monitoring/100.1.0%2Bup19.0.3
+
+The `Alertmanager` is disabled by default in Harvester as of v1.0.2.
+
+https://github.com/harvester/harvester-installer/blob/master/pkg/config/templates/rancherd-13-monitoring.yaml#L33
+
+```
+      alertmanager:
+        enabled: false
+```
+
+The `Alertmanager` will be enabled, and the related features will be described in following stories.
+
+### User Stories
+
+
+#### Story 1
+
+Enable/disable `Alertmanager` on demand or by default.
+
+The `Alertmanager` can be enabled in system installation, then it is available by default.
+
+The `Alertmanager` can also be enabled after system installation, when users want to have it.
```

Comment:
According to the [implementation](https://github.com/harvester/harvester-installer/pull), we can say it's enabled by default and can be turn off later.

Reply:
updated to default enable, thanks.